autogpt_platform/frontend/src/app/layout.tsx

```diff
@@ -22,6 +24,9 @@ export default async function RootLayout({
 }: Readonly<{
   children: React.ReactNode;
 }>) {
+  const headersList = await headers();
+  const host = headersList.get("host") || "";
+  const isPlatformDomain = host === "platform.agpt.co";
```

Comment:
i hate this but i'll be okay -- i really prefer we use the getEnv stuff or centralize it a bit but like htats maybe me being a backend person

Reply:
Agree, I think we should centralize all the env helpers in a single place/service, I'll have a thought 💭